### DISCUSSION TITLE
Homebrew on OpenBSD

### DISCUSSION THREAD

**Discussion by icyrogue** [Homebrew/discussions#3937] [Everyday usage]
Now I know that this question can be a bit crazy, but how hard can it be to port brew on openBSD or BSD OS of any flavour? 

First of all, I know that doing this is probably against openBSD security ideology and overall is a pointless thing to do on server oriented system. I'm interested in porting brew to openBSD because it seems to me like a good challenge even if result is pointless.

So how would you approach porting Homebrew  package manager to openBSD? Where do I even start?

**Comment by SMillerDev** [comment]
I think getting the installer to run would be a start
